agent/wpthook/test_state.cc

```diff
@@ -319,7 +325,14 @@ bool TestState::IsDone() {
   if (_active) {
     bool is_page_done = false;
     CString done_reason;
-    if (test_ms >= _test._minimum_duration) {
+    DWORD elapsed_timeout_ms = ElapsedMsFromTimeoutStart(now);
+    if (_test._overall_timeout_in_ms > 0 &&
+      elapsed_timeout_ms > _test._overall_timeout_in_ms) {

```

Comment:
This should be comparing the _timeout_start_time, not _step_start since it is checking for an overall timeout, right?